pkg/arm/resources.go

```diff
@@ -387,6 +493,9 @@ func Vmss(pc *api.PluginConfig, cs *api.OpenShiftManagedCluster, app *api.AgentP
 		},
 		VirtualMachineScaleSetProperties: &compute.VirtualMachineScaleSetProperties{
 			UpgradePolicy: &compute.UpgradePolicy{
+				AutoOSUpgradePolicy: &compute.AutoOSUpgradePolicy{
+					DisableAutoRollback: to.BoolPtr(false),
```

Comment:
In which case are we going to need a scale set to automatically rollback?

Reply:
this is mirroring a change that the cloud provider appears to make to the VMSS by default - I put it in here to hopefully save the cloud provider finding a diff it needed to make and applying it.